data-plane/config/source/500-dispatcher.yaml

```diff
@@ -61,8 +64,8 @@ spec:
             - mountPath: /etc/config
               name: config-kafka-source-data-plane
               readOnly: true
-            - mountPath: /etc/sources
-              name: kafka-source-sources
+            - mountPath: /etc/contract-resources
+              name: contract-resources
```

Comment:
No volume named `contract-resources` is defined and this is causing problems.

However, I don't get how tests are passed. Maybe there are some "bad" tests here?